themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties

```diff
@@ -0,0 +1,350 @@
+passwordLastUpdateMessage=\u015Eifreniz g\u00FCncellendi
+updatePasswordTitle=\u015Eifre g\u00FCncelle
+updatePasswordMessageTitle=G\u00FC\u00E7l\u00FC bir \u015Fifre se\u00E7ti\u011Finizden emin olun
+updatePasswordMessage=G\u00FC\u00E7l\u00FC bir \u015Fifre, say\u0131lar, harfler ve sembollerin bir kar\u0131\u015F\u0131m\u0131n\u0131 i\u00E7erir. Tahmin etmek zor, ger\u00E7ek bir kelimeye benzemeyen ve sadece bu hesap i\u00E7in kullan\u0131l\u0131r.
+personalSubTitle=Ki\u015Fisel Bilgileriniz
+personalSubMessage=Bu temel bilgileri y\u00F6netin: ad\u0131n\u0131z, soyad\u0131n\u0131z ve e-posta adresiniz
+
+authenticatorCode=Kimlik Do\u011Frulama Kodu
+email=E-Mail
+firstName=Ad
+givenName=Ad
+fullName=Ad Soyad
+lastName=Soyad
+familyName=Soyad
+password=\u015Eifre
+currentPassword=\u015Eimdiki \u015Eifre
+passwordConfirm=\u015Eifre Do\u011Frulama
+passwordNew=Yeni \u015Eifre
+username=Kullan\u0131c\u0131 Ad\u0131
+address=Adres
+street=Cadde
+region=B\u00F6lge
+postal_code=Posta Kodu
+locality=Semt
+country=\u00DClke
+emailVerified=E-Mail Do\u011Fruland\u0131
+gssDelegationCredential=GSS Yetki Bilgisi
+
+profileScopeConsentText=Kullan\u0131c\u0131 profili
+emailScopeConsentText=Email adresi
+addressScopeConsentText=Adres
+phoneScopeConsentText=Telefon numaras\u0131
+offlineAccessScopeConsentText=\u00C7evrimd\u0131\u015F\u0131 Eri\u015Fim
+samlRoleListScopeConsentText=Rollerim
+rolesScopeConsentText=Kullan\u0131c\u0131 rolleri
+
+role_admin=Admin
+role_realm-admin=Realm Admin
+role_create-realm=Realm Olu\u015Ftur
+role_view-realm=Realm g\u00F6r\u00FCnt\u00FCle
+role_view-users=Kullan\u0131c\u0131lar\u0131 g\u00F6r\u00FCnt\u00FCle
+role_view-applications=Uygulamalar\u0131 g\u00F6r\u00FCnt\u00FCle
+role_view-clients=\u0130stemci g\u00F6r\u00FCnt\u00FCle
+role_view-events=Olay g\u00F6r\u00FCnt\u00FCle
+role_view-identity-providers=Identity Provider g\u00F6r\u00FCnt\u00FCle
+role_manage-realm=Realm y\u00F6net
+role_manage-users=Kullan\u0131c\u0131lar\u0131 y\u00F6net
+role_manage-applications=Uygulamalar\u0131 y\u00F6net
+role_manage-identity-providers=Identity Provider y\u00F6net
+role_manage-clients=\u0130stemci y\u00F6net
+role_manage-events=Olay y\u00F6net
+role_view-profile=Profilleri g\u00F6r\u00FCnt\u00FCle
+role_manage-account=Profilleri Y\u00F6net
+role_manage-account-links=Profil ba\u011Flant\u0131lar\u0131n\u0131 y\u00F6net
+role_read-token=Token oku
+role_offline-access=\u00C7evirimd\u0131\u015F\u0131 Yetki
+role_uma_authorization=\u0130zinleri Al
+client_account=M\u00FC\u015Fteri Hesab\u0131
+client_security-admin-console=G\u00FCvenlik Y\u00F6netici Konsolu
+client_admin-cli=Admin CLI
+client_realm-management=Realm-Management
+client_broker=Broker
+
+requiredFields=Zorunlu Alanlar
+allFieldsRequired=T\u00FCm Alanlar Zorunlu
+
+backToApplication=&laquo; Uygulamaya D\u00F6n
+backTo=Geri D\u00F6n {0}
+
+date=G\u00FCn
+event=Olay
+ip=IP
+client=\u0130stemci
+clients=\u0130stemciler
+details=Detaylar
+started=Ba\u015Flang\u0131\u00E7 Tarihi
```

Comment:
If 'Tarihi' is used here (which means the **date**) is should also used for the next 2 lines.